week-1-practise/level-1/1-syntax-error-1.js

```diff
@@ -1,14 +1,16 @@
 // There is a syntax error in this code. Fix it.
 
-function addNumbers(a b c) {
-    return a + b + c;
+function addNumbers(a, b, c) {
+  return a + b + c;
 }
 
 var result = addNumbers(1, 3, 4);
 console.log(result); // Expected result 8 - Run the program and make sure it works
 
-
 // Answer these questions:
 // 1. How many parameters does the function take?
+// The fonction takes three parameters which are a, b, c.
 // 2. What is the function name?
-// 3. Where is the function called? with what parameters?
\ No newline at end of file
+//The function's name is addNumbers.
+// 3. Where is the function called? with what parameters?
+// The function is called outside of the function e.g. : var result = addNumbers();. The parameters are called with these three agreements : 1, 3, 4;
```

Comment:
Good. For the last question, you could have just quoted the line number on which the function is called.